indexer_app/utils.py

```diff
@@ -2,6 +2,8 @@
 import decimal
 import json
 from datetime import date, datetime
+from email.policy import default
+from sys import exception
```

Comment:
What are these?

Reply:
must have been auto imports, not to say i have no idea what they are, lol.
will remove!

Reply:
All good, I removed 👍